.github/workflows/ci.yaml

```diff
@@ -156,16 +168,20 @@ jobs:
       run: echo "BUILD_VERSION=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
       if: startsWith(github.ref, 'refs/tags/')
     - name: Build
-      run: |
-        make publish
+      run: make build
+    - name: Publish
+      run: make publish
     - name: Install
-      run: make install
-    - name: Verify docs
       run: |
-        rm -rf docs/command-reference
-        tanzu apps docs --directory docs/command-reference
-        git diff --exit-code docs
-      continue-on-error: true
+        if [[ ${BUILD_VERSION} ]]; then
+          echo $BUILD_VERSION
+          tanzu plugin install apps --version $BUILD_VERSION --local ./artifacts/published/darwin-amd64
```

Comment:
you can leave the command out of the `if` as it is the same fot the two branches (`if`/`else`)

Reply:
Updated the code to assign build version only if its not set, and tanzu plugin command moved outside